### ISSUE TITLE
Content unreadable

### ISSUE THREAD

**Issue by JacobEvelyn** [randym/axlsx#390]
Hi,

I appear to be getting nondeterministic errors with this gem (I'm using the master branch directly from GitHub). I'm generating a bunch of very basic files with the following code:

```
# HEADER_BACKGROUND_COLOR = "8cd158"

package = Axlsx::Package.new
workbook = package.workbook

workbook.styles do |styles|
  # header_style = styles.add_style(bg_color: HEADER_BACKGROUND_COLOR)

  workbook.add_worksheet(name: "Data") do |sheet|
    sheet.add_row header_row
    # sheet.row_style(0, header_style)

    rows.each { |row| sheet.add_row(row, types: row.map { :string }) }
  end
end
package.use_shared_strings = true
package.serialize(filename)
```

I thought the formatting might be too much so I commented out the lines that are now comments, but I'm still getting errors with ~30-50% of the files. The errors say: `Excel could not open the *.xlsx because some content is unreadable. Do you want to open and repair this workbook?` When I try repairing, the files emerge without any contents at all. The files won't open in Numbers at all, and in OpenOffice they open but are blank.

There's nothing different about the files that are failing that I can discern (all files I'm generating only have data in four cells, and the text contents and length of text is the same between cells in functional files and cells in broken ones). Let me know if there's any other info I can provide!

**Comment by straydogstudio** [comment]
@JacobEvelyn Have you had any luck with this? What rubyzip version are you using?

**Comment by fomichov** [comment]
The same issue here. Rubyzip 1.1.7

**Comment by JacobEvelyn** [comment]
We've removed this part of our code; I'm not sure what version of RubyZip we were using at the time unfortunately, sorry.

**Comment by alepore** [comment]
yeah having the same problem, and i think rubyzip (1.1.7) is not the problem here.
it's super hard to debug these issues, damn Office.

**Comment by alepore** [comment]
for it was just a missing `name` parameter on `#add_worksheet`

**Comment by straydogstudio** [comment]
@alepore So axlsx let you add a worksheet without a name, but that rendered it unopenable?

**Comment by alepore** [comment]
@straydogstudio yes, i did a quick workaround and need to check better but basically this commit (`activeadmin-axlsx`) https://github.com/alepore/activeadmin-axlsx/commit/a131be7b81779e454a8e4561ec6be6c4da7ee9cc solved my MS Excel compatibility problems.
